styles/components/_listingsGrid.scss

```diff
@@ -1,15 +1,65 @@
 .listingsGrid {
```

Comment:
Do you think it's worth it to put a comment in this file that these styles are applied to both the Search and Store pages, so please be mindful of both when making any changes? Your call.